src/main/java/site/coach_coach/coach_coach_server/category/service/CategoryService.java

```diff
@@ -44,4 +42,14 @@ public void deleteCategory(Long routineId, Long categoryId, Long userIdByJwt) {
 			throw new NotFoundCategoryException(ErrorMessage.NOT_FOUND_CATEGORY);
 		}
 	}
+
+	@Transactional
+	public Category changeIsCompleted(Long categoryId, Long routineId) {
+		Category category = categoryRepository.findByCategoryIdAndRoutine_RoutineId(categoryId, routineId)
+			.orElseThrow(() -> new NotFoundCategoryException(ErrorMessage.NOT_FOUND_CATEGORY));
+
+		// Dirty Checking
+		category.setIsCompleted(!category.getIsCompleted());
```

Comment:
category.Completed()

public void completed() {
   this.isCompleted == !this.isCompleted
}